### PR TITLE
Use tagged image in pipeline

### PR DESCRIPTION
**Pull request by bodymindarts**
Move to explicitly tagged genesis image so that pipelines don't break.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 
 ENV SPRUCE_VERSION=1.5.0 \
   CF_CLI_VERSION=6.13.0 \
   VAULT_VERSION=0.6.0 \
   GENESIS_VERSION=1.5.2
 
 # base packages
-RUN apt-get install -yy curl file
+RUN apt-get update \
+      && apt-get install -yy curl file unzip git ruby \
```
Comment (geofffranks) [inline Dockerfile]: @lnguyen  will this get the right version of ruby for bosh_cli, or did we need to do other things to get this working for GE?

**Comment by geofffranks** [comment]
What about just making the pipeline configs use meta.docker_image, and default that to starkandwayne/concourse, overriding as necessary down the road? We have a bunch of non-genesis pipelines with all the same requirements as genesis, so diverging now duplicates the maintenance of that featureset.

If down the road we need to diverge a genesis image, we can base it off starkandwayne/concourse, and add the genesis-specific items?

Or if you prefer, we have a starkandwayne/concourse base image, and starkandwayne/concourse-bosh built off of it, adding all the genesis/boshrelease concourse dependencies?

**Comment by jhunt** [comment]
@bodymindarts any thoughts on @geofffranks' comments?

**Comment by bodymindarts** [comment]
Sounds good to me. The only concern I have is regarding the versioning. Can
we keep starkandwayne/concourse versions in sync with Genesis versions? Or
is it possible to guarantee that Genesis would not pick up a broken image
or one with incompatible deps?

**Comment by jhunt** [comment]
Closing this old PR.

**Pull request closed without merging**